docs/content/install.md

```diff
@@ -69,6 +70,53 @@ You will need to create a [PowerShell Profile][ps-link] if you do not have one.
 iwr "https://cdn.porter.sh/canary/install-windows.ps1" -UseBasicParsing | iex
 ```
 
+# v1 Prerelease
+
+We would love for you to try out v1 prelease and send us any feedback that you have!
+Keep in mind that the v1 prerelease is not suitable for running with production workloads, and that data migrations will not be provided or supported for v1 prerelease.
+The prerelease is intended for you to try out the new features in Porter and provide feedback. It won't work with existing installations.
+
+One way to try out Porter without messing with your current installation of Porter is to install Porter into a different PORTER_HOME directory.
+
+**MacOS**
+
+```bash
+export PORTER_HOME=~/.porterv1
+export VERSION="v1.0.0-alpha.5"
+curl -L https://cdn.porter.sh/$VERSION/install-mac.sh | bash
+```
+
+**Linux**
+
+```bash
+export PORTER_HOME=~/.porterv1
+export VERSION="v1.0.0-alpha.5"
+curl -L https://cdn.porter.sh/$VERSION/install-linux.sh | bash
+```
+
+**Windows**
+
+```powershell
+$PORTER_HOME="$env:USERPROFILE\.porterv1"
+$VERSION="v1.0.0-alpha.5"
+(New-Object System.Net.WebClient).DownloadFile("https://cdn.porter.sh/$VERSION/install-windows.ps1", "install-porter.ps1")
+.\install-porter.ps1 -PORTER_HOME $PORTER_HOME
+```
+
+Now when you want to use the v1 version of Porter, set the PORTER_HOME environment variable and add it to your PATH.
+
```

Comment:
I'm confused by the setting of environment variables (other than for the Windows instructions). Is there a missing step or is it assumed that they are in the right directory to start with?

Reply:
Thanks for calling that it, I see now that I left off how to use porter after installing, i.e. how to reset the env vars each time you want to switch to a different version of porter.